### PR TITLE
Rkaruppaiah get schedule

### PR DESCRIPTION
**Pull request by tavisca-rkaruppaiah**
Please review get details of the schedule @tavisca-sgakhar

### CODE REVIEW

File: Tavisca.TMS.ApiTests/AppConfig.cs [hunk 1]
```diff
@@ -6,6 +6,6 @@ namespace Tavisca.TMS.ApiTests
 {
     public class AppConfig
     {
-        public const string BaseUrl = @"https://localhost:5001/api";
+        public const string BaseUrl = @"https://localhost:44356/api";
     }
```
Comment (sdesai-tavisca) [inline Tavisca.TMS.ApiTests/AppConfig.cs]: Why changed this port?

File: src/Tavisca.TMS.Core/Services/ScheduleService.cs [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+                    }
+                }
+                var result = await _repository.CreateSchedule(id, schedule);
+                if(result)
+                {
+                    return new ApiSuccessResponse("Schedule created successfully.");
+                }
+                else
+                {
+                    return new ApiFailureResponse(ErrorCodes.MySqlException, new List<ErrorInfo> { new ErrorInfo(ErrorCodes.MySqlException, ErrorMessages.MySqlException) });
```
Comment (sdesai-tavisca) [inline src/Tavisca.TMS.Core/Services/ScheduleService.cs]: Change this error message type to user understandable format

File: src/Tavisca.TMS.Core/Services/ScheduleService.cs [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+                List<ErrorInfo> errors = GetErrorListForSchedule(validationResult);
+                return new ApiFailureResponse(ErrorCodes.InValidScheduleDetails, errors);
+            }
+           
+            
+        }
+
+        private List<ErrorInfo> GetErrorListForSession(ValidationResult validationResult)
+        {
+            List<ErrorInfo> errors = new List<ErrorInfo>();
```
Comment (sdesai-tavisca) [inline src/Tavisca.TMS.Core/Services/ScheduleService.cs]: Change switch case implementation

File: src/Tavisca.TMS.Core/Services/ScheduleService.cs [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+                    }
+                }
+                var result = await _repository.UpdateSchedule(id, schedule);
+                if (result)
+                {
+                    return new ApiSuccessResponse("Schedule updated successfully.");
+                }
+                else
+                {
+                    return new ApiFailureResponse(ErrorCodes.MySqlException, new List<ErrorInfo> { new ErrorInfo(ErrorCodes.MySqlException, ErrorMessages.MySqlException) });
```
Comment (sdesai-tavisca) [inline src/Tavisca.TMS.Core/Services/ScheduleService.cs]: Change error message

File: src/Tavisca.TMS.Core/Services/ScheduleService.cs [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+        }
+
+
+        private List<ErrorInfo> GetErrorListForSchedule(ValidationResult validationResult)
+        {
+            List<ErrorInfo> errors = new List<ErrorInfo>();
+            for (int i = 0; i < validationResult.Errors.Count; i++)
+            {
+
+                switch (validationResult.Errors[i].ErrorMessage)
```
Comment (sdesai-tavisca) [inline src/Tavisca.TMS.Core/Services/ScheduleService.cs]: change switch case implementation

File: src/Tavisca.TMS.Persistence/Dal/ScheduleDal.cs [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+            }
+            return result;
+        }
+
+        private async Task<ScheduleDto> GetAllDetails(MySqlDataReader reader)
+        {
+            ScheduleDto scheduleDto = new ScheduleDto();
+            await Task.Run(()=> 
+            {
+                scheduleDto.Name = reader["name"].ToString();
```
Comment (sdesai-tavisca) [inline src/Tavisca.TMS.Persistence/Dal/ScheduleDal.cs]: null check should be there

**Comment by tavisca-rkaruppaiah** [comment]
@ckulkarni007 can you check

**Pull request merged**